pdf_parser/pdf_parse.py

```diff
@@ -42,13 +38,24 @@ def parse_pdf_document(document):
         )
         return None, None
 
+    try:
+        # Try to get file stats in order to check both its existance
+        # and if it has some content
+        st = os.stat(parsed_path)
+
```

Comment:
Thanks a ton! I know that's a bit fussy to do. Good to check the empty file though.